app/assets/javascripts/modules/remove-filter.js

```diff
@@ -57,9 +57,12 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
         var haystack = ' ' + currentVal.trim() + ' '
         var needle = ' ' + decodeEntities(removeFilterValue.toString()) + ' '
         var newVal = haystack.replace(needle, ' ').replace(/ {2}/g, ' ').trim()
-        window.GOVUK.triggerEvent($input.val(newVal)[0], 'change', { detail: { suppressAnalytics: true } })
+        $input.value = newVal
+        window.GOVUK.triggerEvent($input, 'change', { detail: { suppressAnalytics: true } })
       } else if (elementType === 'OPTION') {
-        window.GOVUK.triggerEvent($('#' + removeFilterFacet).val('')[0], 'change', { detail: { suppressAnalytics: true } })
+        var element = document.getElementById(removeFilterFacet)
```

Comment:
Are we sure this element will always exist? should we have a conditional so we don't try set value on null

Reply:
As far as I've been able to manually test yes. This has to exist for the thing to be clicked in the first place. It does cause errors if it doesn't exist (as with your other comment) which is why i had to add [this commit](https://github.com/alphagov/finder-frontend/pull/2556/commits/7cc4280cb4da67f89a6d89f018a826f06a322d2d) but I haven't seen it when spinning the app up locally and the tests don't suggest that it can happen.